nmt/utils/nmt_utils.py

```diff
@@ -61,7 +61,7 @@ def decode_and_evaluate(name,
                 sent_id,
                 tgt_eos=tgt_eos,
                 bpe_delimiter=bpe_delimiter)
-            trans_f.write("%s\n" % translation)
+            utils.print_out(translation, f=trans_f)
```

Comment:
I think this is fixed. Can you try revert this and rebase with the head to see if that resolves your problem. We also don't want to print out all translation to stdout.